yarn/src/main/scala/org/apache/spark/deploy/yarn/security/CredentialUpdater.scala

```diff
@@ -53,50 +53,58 @@ private[spark] class CredentialUpdater(
       override def run(): Unit = Utils.logUncaughtExceptions(updateCredentialsIfRequired())
     }
 
-  @volatile private var timeOfNextUpdate = sparkConf.get(CREDENTIALS_UPDATE_TIME)
+  /** Start the credentail updater thread periodically to get the credentials */
```

Comment:
"credential"

Also the whole comment sounds redundant. Just: "Start the credential updater task."